examples/plotter/Cargo.toml

```diff
@@ -15,11 +15,23 @@ name = "plotter"
 
 [dependencies]
 slint = { path = "../../api/rs/slint" }
-plotters = { version = "0.3.1", default-features = false, features = ["bitmap_backend", "surface_series"] }
+plotters = { git = "https://github.com/Be-ing/plotters.git", branch = "font-kit-update", default-features = false, features = ["bitmap_backend", "surface_series"] }
```

Comment:
Would you be okay with merging this before https://github.com/38/plotters/pull/337 is merged upstream? The plotters maintainer hasn't made any commits in 6 months. This PR has been open for a while and I'd like to stop rebasing it periodically. I think this would be okay considering this is only used in an example but if you want to wait until it is merged upstream, I'd understand.

Reply:
Yes, that's ok, but please leave a comment stating the reason of using a git URL.

Reply:
This is no longer required now that the maintainer of plotters has returned and added more maintaners.